### DISCUSSION TITLE
Playback stuttering with 120fps videos

### DISCUSSION THREAD

**Discussion by Lumiukko** [mifi/lossless-cut#2036] [Support] [answered]
I'm having some issues playing 120fps files (it's only 1080p120), HEVC hardware support on/off doesn't appear to help.
Is there a way to improve this?

Changing the FPS to 60 helps with the playback, but makes examination of all the material a little slow :)
I also always have to remember changing back to 120fps, otherwise the exported files will be only at 50% speed.

**Comment by mifi** [comment, marked as answer]
Windows? Could it possibly be due to one of https://github.com/mifi/lossless-cut/issues/922 https://github.com/mifi/lossless-cut/issues/1904 https://github.com/mifi/lossless-cut/issues/1915 https://github.com/mifi/lossless-cut/issues/922 ?

**Reply by Lumiukko** [reply]
OMG, the #1915 worked, i.e. using Vista SP2 compatibility mode in Windows. What is this sorcery?! 

Thank you so much :)
